site/profile/manifests/cvmfs.pp

```diff
@@ -10,16 +11,31 @@
     source   => 'https://ecsft.cern.ch/dist/cvmfs/cvmfs-release/cvmfs-release-latest.noarch.rpm'
   }
 
-  package { 'cc-cvmfs-repo':
-    ensure   => 'installed',
-    provider => 'rpm',
-    name     => 'computecanada-release-1.0-1.noarch',
-    source   => 'https://package.computecanada.ca/yum/cc-cvmfs-public/prod/RPM/computecanada-release-latest.noarch.rpm'
+  if $::software_stack == 'eessi' {
+    package { 'stack':
+      ensure   => 'installed',
+      provider => 'rpm',
+      name     => 'cvmfs-config-eessi-0.2.3-1.noarch',
+      source   => 'https://github.com/EESSI/filesystem-layer/releases/download/v0.2.3/cvmfs-config-eessi-0.2.3-1.noarch.rpm'
```

Comment:
@bedroge We were also thinking of having a moving tag "latest" here, maybe this is a good motivation for it?

Reply:
Definitely (besides a bunch of others)! With a real yum repo, which we were also considering, this could also be solved easily. But maybe we could do this (`latest` tag) in the meantime.

Reply:
Unfortunately, puppet `package` name attribute has to match exactly the name of the rpm (including the version). So unless, the version is set to `latest` in the RPM spec file, while we could install from a source that has `latest` in its rpm name, we would still have to set the explicit version number in the `name` field. If we don't, puppet will install the package correctly, but will fail to correctly identify the package was installed and will try to install it again at each puppet run.

There is an example in the [globus.pp file line 3-8](https://github.com/ComputeCanada/puppet-magic_castle/blob/master/site/profile/manifests/globus.pp#L3). Everytime Globus release a new version, I have to update the version number.

If you are considering real yum repo, I recommend looking at Fedora COPR - http://copr.fedorainfracloud.org/. This is what I used to build a yum repo for Slurm, singularity, and other projects that are used by Magic Castle, but do not provide a yum repo.